src/DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -271,14 +271,11 @@ public function process(ContainerBuilder $container): void
             }
         }
 
-        if (!empty($customTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setTypes', [$customTypes]);
-        }
-        if (!empty($customNotMappedTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setNotMappedTypes', [$customNotMappedTypes]);
-        }
+        $staticMapperDefinition = new Definition(StaticTypeMapper::class, [
+            '$types' => $customTypes,
+            '$notMappedTypes' => $customNotMappedTypes
+        ]);
+        $container->setDefinition('graphql.type_mapper', $staticMapperDefinition);
```

Comment:
it is necessary to preserve tag to assure that mapper will be used for webonyx schema

```suggestion
        $staticMapperDefinition->addTag('graphql.type_mapper');
        $container->setDefinition('graphql.type_mapper', $staticMapperDefinition);
```